### ISSUE TITLE
find heavily used tools with large runtime

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/gxadmin#126]
I would like to start optimizing some tools. I think we can find some BA and MA students to optimize our tool stack and with that waste fewer resources.

To prepare a query like `gxadmin query tool-usage` but with a min/avg/max runtime column would be super useful.

**Comment by hexylena** [comment]
you've fixed the cgroup data we were getting? That's exciting to hear. How did that happen? Last time I looked at it some of the cgroup data was a bit suspect, to say the least.

**Comment by bgruening** [comment]
Nope, not fixed I think, they are still very flaky. But here we only need runtimes. An estimate of 1h+ is enough I would say to rank a few tools top. I suspect all the galaxy_sequence_utils popping up top, or some small python/perl scripts ... groomer etc ... hopefully ... those would be also low-hanging fruits to make them faster.

**Comment by hexylena** [comment]
damn, I was hopeful. Makes sense, good luck!
